defold-spine/commonsrc/spine_loader.cpp

```diff
@@ -208,6 +209,25 @@ namespace dmSpine
         return &regions[*anim_index];
     }
 
+    static int /*bool*/ loadSequence(dmHashTable64<uint32_t>* name_to_index, spAtlasRegion* atlasRegions, const char *basePath, spSequence *sequence, spAtlasRegion* default_region) {
+        bool is_atlas_available = name_to_index != 0;
+
+        spTextureRegionArray *regions = sequence->regions;
+        char *path = CALLOC(char, strlen(basePath) + sequence->digits + 1);
+        int i;
+        for (i = 0; i < regions->size; i++) {
+            spSequence_getPath(sequence, basePath, i, path);
+            regions->items[i] = is_atlas_available ? SUPER(FindAtlasRegion(name_to_index, atlasRegions, path)) : SUPER(default_region);
+            if (!regions->items[i]) {
+                FREE(path);
+                return 0;
+            }
+            regions->items[i]->rendererObject = regions->items[i];
+        }
+        FREE(path);
+        return -1;
```

Comment:
While it's true that bools are generally "0 == false", and "not 0 == true", it is also most common to have positive values for true values.

Since it is our own function, we should return 1 here.
Or even, since it's C++, return an actual bool.

Reply:
It makes sense!